lib/pages/list.js

```diff
@@ -64,6 +69,7 @@ module.exports = React.createClass({displayName: 'ListPage',
 
             <h2>Owned by you:</h2>
             {owned}
+        <PrSearcher controller={this.props.controller} history={this.props.history} />
```

Comment:
I will fiddle with this later, as it should probably be part of a consistent header on all pages